frontend/src/views/Schools.svelte

```diff
@@ -46,11 +47,26 @@
       <ul class="list-group w-100">
         {#each schools as school}
           <li class="list-group-item d-flex justify-content-between align-items-center">
+            {#if school.isServiceEnabled}
```

Comment:
It seems the if/else code has indentation issues? Do you have prettier running automatically in your VSCode?